services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/oauth/OpenIDConnectManagerImpl.java

```diff
@@ -179,8 +190,29 @@ public OIDCAuthorizationRequestDescription getAuthenticationRequestDescription(
 		}
 		return scopeDescriptions;
 	}
+	
+	static String getScopeHash(OIDCAuthorizationRequest authorizationRequest) {
```

Comment:
Check if this static method can go into a utility class that already exist (if any)